examples/ui/ui.rs

```diff
@@ -78,6 +83,20 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
                                 // for accessibility to treat the text accordingly.
                                 Label,
                             ));
+
+                            #[cfg(feature = "bevy_ui_debug")]
+                            // Debug overlay text
+                            parent.spawn((
+                                TextBundle::from_section(
+                                    "Press Space to enable debug outlines.",
+                                    TextStyle {
+                                        font: asset_server.load("fonts/FiraSans-Bold.ttf"),
+                                        font_size: 20.,
+                                        ..Default::default()
+                                    },
+                                ),
+                                Label,
+                            ));
```

Comment:
you could display a text if the feature is not enabled to say "try enabling feature `bevy_ui_debug`"